doc/source/ray-contribute/development.rst

```diff
@@ -343,9 +343,9 @@ You can tweak the build with the following environment variables (when running `
   python packages
 - ``RAY_DEBUG_BUILD``: Can be set to ``debug``, ``asan``, or ``tsan``. Any
   other value will be ignored
-- ``BAZEL_LIMIT_CPUS``: If set, it must be an integers. This will be fed to the
-  ``--local_cpu_resources`` argument for the call to Bazel, which will limit the
-  number of CPUs used during Bazel steps.
+- ``BAZEL_ARGS``: If set, pass a space-separated set of arguments to Bazel. This can be useful
+  for restricting resource usage during builds, for example. See https://bazel.build/docs/user-manual
+  for more information about valid arguments.
```

Comment:
could you keep `BAZEL_LIMIT_CPUS` for backwards compatibility, just in case people already have integrations using them? basically, just add `BAZEL_ARGS` but do not need to remove `BAZEL_LIMIT_CPUS`.

if you want to go fancier, maybe print a warning to stderr when `BAZEL_LIMIT_CPUS` is set.

thanks.

Reply:
Sounds good. I've added back in the old option, and included a warning to users to switch to using `BAZEL_ARGS` using the same language `numpy` uses for deprecation warnings.